src/main/scala/firrtl/passes/Checks.scala

```diff
@@ -336,9 +336,9 @@ object CheckTypes extends Pass {
         case (e: WSubIndex) => e.exp.tpe match {
           case (t: VectorType) if e.value < t.size =>
           case (t: VectorType) =>
-            errors append (new IndexTooLarge(info, mname, e.value))
+            errors append new IndexTooLarge(info, mname, e.value)
```

Comment:
Rather than removing the parentheses we should actually shift these to `.` notation since they are modifying a mutable data structure